apps/web/pages/api/auth/forgot-password.ts

```diff
@@ -28,7 +28,8 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     });
 
     if (!maybeUser) {
-      return res.status(400).json({ message: "Couldn't find an account for this email" });
+      // Don't leak information about whether an email is registered or not
+      return res.status(200).json({ message: "Reset Requested" });
```

Comment:
On FE still redirect to success screen - even If the user isn't found

Reply:
good

Reply:
Something like`If this email exists in our system, you should receive a Reset email.` is more accurate without leaking information.